### PR TITLE
Disable cookie list on production

### PR DESCRIPTION
**Pull request by antonok-edm**
As per https://bravesoftware.slack.com/archives/C2FQMN4AD/p1649888650104789?thread_ts=1649864731.514619&cid=C2FQMN4AD

### CODE REVIEW

**Review by mihaiplesa — APPROVED** [review]

**Review by moritzhaller — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `production` approved after deliberating with @brave/uplift-approvers.

**Pull request merged**
